src/components/Playbooks/Runs/Actions/PlaybookRunsActions.tsx

```diff
@@ -67,7 +68,12 @@ export default function PlaybookRunsActions({ data }: PlaybookRunActionsProps) {
             )}
           <VerticalDescription
             label="Duration"
-            value={<Age from={data.start_time} to={data.end_time} />}
```

Comment:
I think this also applies to the per step duration  - can you also check for other durations